packages/synapse-react-client/test/lib/containers/dataaccess/AccessHistoryDashboard.test.tsx

```diff
@@ -24,34 +25,25 @@ import {
 } from '../../../../mocks/user/mock_user_profile'
 import selectEvent from 'react-select-event'
 import { getOptionLabel } from '../../../../src/lib/containers/dataaccess/AccessRequirementSearchBox'
-
+import * as AccessApprovalsTableModule from '../../../../src/lib/containers/AccessApprovalsTable'
+import * as AccessRequestSubmissionTableModule from '../../../../src/lib/containers/AccessRequestSubmissionTable'
+import { UserBundle } from '../../../../src/lib/utils/synapseTypes'
 const APPROVAL_TABLE_TEST_ID = 'AccessApprovalTableTestId'
 const SUBMISSION_TABLE_TEST_ID = 'AccessSubmissionTableTestId'
 
-jest.mock('../../../../src/lib/containers/AccessApprovalsTable', () => ({
-  AccessApprovalsTable: jest.fn().mockImplementation(() => {
+const mockAccessApprovalsTable = jest
+  .spyOn(AccessApprovalsTableModule, 'AccessApprovalsTable')
```

Comment:
Prefer more type-safe and less verbose `spyOn` over `mock`

Reply:
Noted!